src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Let's specify what exactly string can it be
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/a467cdd7-2d1d-4ab6-8339-0241fa2abb9e)